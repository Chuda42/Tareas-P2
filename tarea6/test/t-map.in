# prueba de tiempo mapping
tiempo_map 1000000
Fin
