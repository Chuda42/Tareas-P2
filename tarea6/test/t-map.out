1>#  prueba de tiempo mapping.
2>prueba asociar, desasociar, existeAsociacion. 
prueba existeAsociacion, valorEnMap. 
3>Fin.
